ultimate_discord_nuke.py

```diff
@@ -1,243 +1,53 @@
 import discord
 import asyncio
-from discord.ext import commands
 import time
+import sys
+from discord.ext import commands
 from random import randint
 
-bot = commands.Bot(command_prefix = "!")
+intents = getattr(getattr(discord, "Intents"), "default")()
+setattr(intents, "members", True)
+
+bot = commands.Bot(command_prefix="!", intents=intents)
```

Comment:
WHY DID YOU NAME IT BOT WTF
proper naming scheme is `ctx` or `balls` or `globals` or `discord` or `discord_js`